content/docs/command-reference/move.md

```diff
@@ -17,27 +17,29 @@ positional arguments:
 ## Description
 
 `dvc move` is useful when a `src` file or directory has previously been added to
-the <abbr>project</abbr> with `dvc add`, creating a `.dvc` file (with `src` as a
-dependency). `dvc move` behaves like `mv src dst`, moving `src` to the given
-`dst` path, but it also renames and updates the corresponding `.dvc` file
-appropriately.
+the <abbr>project</abbr> with `dvc add` or `dvc import`, creating a `.dvc` file
+(with `src` as a dependency). `dvc move` doesn't support renaming stage
+<abbr>outputs</abbr> (see `dvc.yaml`), they have to be renamed manually.
 
-> Note that `src` may be a copy or a
+> Note that `src` may be either a
 > [link](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
-> to a file in cache. The cached file is not changed by this command.
+> or a copy to the corresponding data in the cache. The cached file is not
+> changed by this command.
 
-If the destination path (`dst`) already exists and is a directory, the source
-code file or directory (`src`) is moved unchanged into this folder along with
-the corresponding `.dvc` file.
+`dvc move` behaves like `mv src dst`, moving `src` to the given `dst` path, but
+it also renames and updates the corresponding `.dvc` file appropriately. If the
+destination path (`dst`) is a directory and already exists, the source code file
+or directory (`src`) is moved unchanged into this folder along with the
+corresponding `.dvc` file.
 
 Let's imagine the following scenario:
 
 ```dvc
 $ dvc add data.csv
 ```
 
-The `dvc add` command would create a `data.csv.dvc` `.dvc` file with the
-following content:
+The `dvc add` command would create a `.dvc` file named as `data.csv.dvc` with
```

Comment:
"name as" ?

Reply:
`.dvc` is repeating that's why I changed it.